keepalived-vip/controller.go

```diff
@@ -284,12 +302,13 @@ func (ipvsc *ipvsControllerController) Stop() error {
 }
 
 // newIPVSController creates a new controller from the given config.
-func newIPVSController(kubeClient *unversioned.Client, namespace string, useUnicast bool, configMapName string, vrid int) *ipvsControllerController {
+func newIPVSController(kubeClient *unversioned.Client, namespace string, useUnicast bool, configMapName string, vrid int, dryrun bool) *ipvsControllerController {
```

Comment:
Please find a different way to mock the IPVS controller, rather than interleaving the logic as a "dryrun" flag